dangerfile.js

```diff
@@ -111,8 +111,8 @@ if (componentChangedOrCreated) {
 
 // If any of the packages have changed, the UNRELEASED log should have been updated.
 const unreleasedModified = includes(modifiedFiles, 'UNRELEASED.yaml');
-const packagesModified = fileChanges.some(filePath =>
-  filePath.startsWith('packages/'),
+const packagesModified = fileChanges.some(
+  filePath => filePath.startsWith('packages/') && !filePath.endsWith('.md'),
```

Comment:
Happy to undo this change, but I think it makes sense to avoid leaving the message `One or more packages have changed, but UNRELEASED.yaml wasn't updated.` when only `*.md` files have been modified :-)